frontend/src/gen/vultisig/keysign/v1/keysign_message_pb.ts

```diff
@@ -29,6 +29,12 @@ import { THORChainSwapPayload } from './thorchain_swap_payload_pb.js';
 import { OneInchSwapPayload } from './1inch_swap_payload_pb.js';
 import { Erc20ApprovePayload } from './erc20_approve_payload_pb.js';
 
+export enum SwapPayloadType {
```

Comment:
Please don't add stuff here , because this file is generated automatically by protobuf , it will be wipe out next time when we generate it again